server/src/test/java/org/opensearch/common/settings/ScopedSettingsTests.java

```diff
@@ -1462,4 +1463,21 @@ public List<String> getListValue(final List<String> value) {
         );
     }
 
+    @Test
```

Comment:
Gradle precommit is failing on this annotation. Please remove it.  Simply naming the method `test...()` is enough.